custom/icds/location_reassignment/models.py

```diff
@@ -64,8 +64,9 @@ def perform(self):
         new_locations_created = self._create_missing_new_locations()
         self.operation_obj.new_locations.extend(new_locations_created)
         self.operation_obj.perform()
-        for old_location in self.operation_obj.old_locations:
-            deactivate_users_at_location(old_location.location_id)
+        if self.operation != EXTRACT_OPERATION:
```

Comment:
What would you think of making a flag like `Transition.should_archive_users`? I think it can be a nice kind of documentation for someone reading through the classes.

Reply:
Sounds good.
https://github.com/dimagi/commcare-hq/pull/27344/commits/6f3407bde6d713d89ad4c3a0abaf0a1fb9c94cf9